### ISSUE TITLE
-g header

### ISSUE THREAD

**Issue by caiquejjx** [defnotcat/libag3-c#5]
Hey defnotcat, it's me again bothering you.
Your help was great to me, I was able to get everything I needed on the `-e` header but now I'm struggling with the `-g` one and I was wondering if you saw anything about it?
I was able to track most functions but I reached a part where they seem to use `LuaJIT 2.1.0-beta3` which as the name suggests it's a JIT for LUA and I even got to see some code they are passing to the Lua part but I have no idea how they do it since there's no lua files or anything related to lua in the project and there's a point where tracing and hooking functions is not helping anymore so I appreciate any tips from you.

Thanks again

**Comment by defnotcat** [comment]
Hey there! I'm glad to hear that I could be helpful, unfortunately I have close to zero experience with the `-g` header, I did come across it when browsing the library but I quickly discarded it as it seemed pretty involved with the web view and the other headers that come from it. 

I basically ignored all headers but `-e` when I made this, because it was the only one that had very few references (1-2 iirc) to stuff related to the web view, but also because providing only this header used to be enough to pass the check, but I suppose that's not the case anymore.

Sorry I couldn't be of more help with this one, good luck!
